codegen/runner/runner.go

```diff
@@ -49,6 +49,24 @@ func checkError(err error, message string) {
 	}
 }
 
+type codegenOptions struct {
```

Comment:
nit: can we group these into some cohesive fashion, if not, just alphabetize? will probably come later?

nit: link to docs describing these options?